stacks/PickupStack.ts

```diff
@@ -62,6 +63,14 @@ export function PickupStack ({ app, stack }: StackContext): void {
       // visibilityTimeout: Duration.minutes(5),
       // for debug!
       enableExecuteCommand: true,
+      healthCheck: {
+        command: ['CMD-SHELL', 'ps -ef | grep node || exit 1'],
```

Comment:
My only concern with this health check is that it might come back showing yes if other processes have node in the name and that's a common-ish word in clusters/infrastructure.  Is there any pickup-specific string we could be searching for here?

It looks like the [docker image for pickup](https://github.com/web3-storage/pickup/blob/c7cbfe834f83f832fbef7312a3619a9fb0da83d0/Dockerfile#L10) runs `node start -w pickup` so maybe we can just grep for "pickup" instead?

Reply:
Agreed, changed to get pickup instead node process